extract.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# Authors:
+# 2017 Youfu Zhang
+# 2019 Balint Reczey <balint.reczey@canonical.com>
+
+set -e
+
+usage() {
+    echo "Usage: $0 < Unifi Controller backup file > < output directory >"
```

Comment:
I would prefer `<UniFi Controller backup file>` than `< Unifi Controller backup file >`. Without extra spaces, `UniFi` instead of `Unifi`.